school/person.rb

```diff
@@ -1,3 +1,4 @@
+require_relative 'decorate'
 class Person
```

Comment:
- [ ] From the project requirements, the Person class is supposed to be inherited from the Nameable class 👇 
![image](https://user-images.githubusercontent.com/90222110/194087272-5ad47a24-2164-4f1f-baa7-ce6e124bcdf9.png)

In addition, it should define a `correct_name` method that returns the name attribute. Please implement it 😃 👍

HINT 💡 
```
class Person < Nameable
   ...

  def correct_name
      name
  end
end
```